src/platforms/hosted/jlink.c

```diff
@@ -67,7 +67,7 @@ int jlink_simple_request(const uint8_t command, const uint8_t operation, void *c
 }
 
 /*
- * This runs JLINK_CMD_IO_TRANSACT transactions, these have the following format:
+ * This runs JLINK_CMD_HW_JTAG3 transactions, these have the following format:
```

Comment:
While this is technically correct, what "HW_JTAG3" means is less clear - which is why we'd specifically renamed it for BMDA. A note about how our command names map to the manual might be a good call, but we would prefer to keep the more descriptive naming (it's performing an IO transaction!).

Same for the naming of things like the reset commands. We'd done that for code clarity because we got fed up of having headaches trying to understand the J-Link code every time we dug into it and needing the manual open to understand the code.